OpenRA.Mods.RA/Traits/Disguise.cs

```diff
@@ -115,7 +115,14 @@ void DisguiseAs(Actor self, Actor target)
 				var tooltip = target.TraitsImplementing<IToolTip>().FirstOrDefault();
 				AsTooltipInfo = tooltip.TooltipInfo;
 				AsPlayer = tooltip.Owner;
```

Comment:
It would make sense to move this into the else branch for symmetry and to avoid double-assigning it.